### PR TITLE
[RLlib] Deflake cartpole crashing tests.

### PR DESCRIPTION
**Pull request by gjoliver**
## Why are these changes needed?

Make sure cartpole crashing tests are not flaky.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by krfricke — APPROVED** [review]
Looks good to me, do we need to eval with crashing env at all?

**Comment by gjoliver** [comment]
I don't feel like we should. let's wait for CI and merge. thanks.

File: rllib/tuned_examples/pg/cartpole-crashing-pg.yaml [hunk 1]
```diff
@@ -30,4 +35,12 @@ cartpole-crashing-pg:
         evaluation_duration_unit: episodes
         evaluation_parallel_to_training: true
         evaluation_config:
-          explore: false
+            explore: false
+            env_config:
+                config:
+                    # Make eval workers solid.
+                    # This test is to prove that we can learn with crashing env,
```
Comment (kouroshHakha) [inline rllib/tuned_examples/pg/cartpole-crashing-pg.yaml:42]: Do you need disable_env_checking even with these numbers that forces the env not to crash?

Comment (gjoliver) [reply rllib/tuned_examples/pg/cartpole-crashing-pg.yaml:42]: yeah I do. env can crash during the actual checking part of the env_checker, which will then crash the entire test.

**Pull request merged**